chapters/en/module3.md

```diff
@@ -258,7 +258,7 @@ Have you include the README.md file in the staging area?
 </exercise>
```

Comment:
I think you missed one above `<exercise id='7' title='Committing changes to a local repository' type='slides, video'>` does not have the correct video added.